metrics/web/lib/auth/presentation/state/auth_notifier.dart

```diff
@@ -43,8 +52,16 @@ class AuthNotifier extends ChangeNotifier {
   /// Determines if a user is authenticated.
   bool get isLoggedIn => _isLoggedIn;
 
-  /// Returns an AuthErrorMessage, containing an auth error message.
-  AuthErrorMessage get authErrorMessage => _authExceptionDescription;
+  /// Returns an [AuthErrorMessage], containing an authentication error message.
+  AuthErrorMessage get authErrorMessage => _authErrorMessage;
+
+  /// Returns an [EmailValidationErrorMessage], containing an email validation error message.
+  EmailValidationErrorMessage get emailValidationErrorMessage =>
+      _emailValidationErrorMessage;
+
+  /// Returns an [PasswordValidationErrorMessage], containing a password validation error message.
+  PasswordValidationErrorMessage get passwordValidationErrorMessage =>
+      _passwordValidationErrorMessage;
```

Comment:
Maybe for UI we should return String - to make sure that it's close to ViewModel approach?